maykin_common/settings.py

```diff
@@ -54,6 +54,14 @@
 Required for the :ref:`quickstart_pdf` extra.
 """
 
+LOGIN_URLS = []
+"""
+List of login URLs.
```

Comment:
```suggestion
Collection of login URLs.
```
A list, tuple or set would all work :)